Monika After Story/game/script-songs.rpy

```diff
@@ -740,4 +740,89 @@ label mas_song_hero_long:
     m 6dku "{i}~That a hero lies in you~{/i}"
     m 4eua "{i}~That a hero lies in you~{/i}"
     return
+    
+init 5 python:
+    addEvent(
+        Event(
+            persistent._mas_songs_database,
+            eventlabel="mas_song_our_reality",
+            prompt="Our Reality",
+            category=[store.mas_songs.TYPE_LONG],
+            unlocked=False,
+            aff_range=(mas_aff.NORMAL, None)
+        ),
+        code="SNG"
+    )
 
+label mas_song_our_reality:
+    if persistent.gender == "F":
+        $ gen = "her"
+    elif persistent.gender == "M":
+        $ gen = "his"
+    else:
+        $ gen = "their"
+
+    window hide
```

Comment:
Ideally would need some lead in dialogue here. It's kinda abrupt to just slide offscreen after coming here from the pick song menu.

Ideally you'd just need to add a label parameter which you pass in where you call it from `monika_credits_song` to skip this dialogue.